File: .claude/commands/unit-test-writer/tests/fixtures/python/src/calculator.py
def add(a: float, b: float) -> float:
    return a + b


def subtract(a: float, b: float) -> float:
    return a - b


def multiply(a: float, b: float) -> float:
    return a * b


def divide(a: float, b: float) -> float:
    if b == 0:
        raise ZeroDivisionError("Cannot divide by zero")
    return a / b


def power(base: float, exp: int) -> float:
    if exp < 0:
        raise ValueError("Negative exponents not supported")
    result = 1.0
    for _ in range(exp):
        result *= base
    return result

File: .claude/commands/unit-test-writer/tests/fixtures/go/calculator.go
package calculator

import "errors"

func Add(a, b float64) float64 {
	return a + b
}

func Subtract(a, b float64) float64 {
	return a - b
}

func Multiply(a, b float64) float64 {
	return a * b
}

func Divide(a, b float64) (float64, error) {
	if b == 0 {
		return 0, errors.New("division by zero")
	}
	return a / b, nil
}

func Power(base float64, exp int) (float64, error) {
	if exp < 0 {
		return 0, errors.New("negative exponents not supported")
	}
	result := 1.0
	for range exp {
		result *= base
	}
	return result, nil
}
